content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,217 @@
+  maintenance, installation, repair)
+
+.. image:: type/product-form.png
+   :align: center
+   :alt: Set a product type on the product form.
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, install the corresponding apps **in addition** to *Inventory*.
+
+   To install additional applications, navigate to :menuselection:`Apps` and select the
+   :guilabel:`Activate` button to install the desired apps or modules.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Calendar* and
+  *Appointment* apps. Requires the :guilabel:`Pay to Book` module to be installed
+- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *POS* app
+  installed
+- :guilabel:`Event Ticket`: sell tickets to attend an event. Requires installation of the *Events*
+  app
+- :guilabel:`Event Booth`: charge customers for setting up a booth. Requires installation of the
+  *Events* app
+- :guilabel:`Course`: sell access to an educational course. Requires installation of the *eLearning*
+  app.
+
+Compare types
+=============
+
+Below is a summary of how each product type affects common *Inventory* operations, like transfers,
+reordering rules, and the forecasted report. Click the chart item with an asterisk(*) to navigate to
+detailed sections.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Product type
+     - Storable
+     - Consumable
+     - Service
+   * - Physical product
+     - Yes
+     - Yes
+     - No
+   * - On-hand quantity
+     - :ref:`Yes* <inventory/product_management/on-hand-store>`
+     - :ref:`Yes* <inventory/product_management/on-hand-con>`
+     - No
+   * - :doc:`Inventory valuation
+       <../../warehouses_storage/inventory_valuation/using_inventory_valuation>`
+     - Yes
+     - No
+     - No
+   * - Create transfer
+     - :ref:`Yes* <inventory/product_management/transfer-store>`
+     - :ref:`Yes* <inventory/product_management/transfer-con>`
+     - :ref:`No* <inventory/product_management/transfer-serv>`
+   * - :doc:`Lot/serial number tracking <differences>`
+     - Yes
+     - No
+     - No
+   * - Appears on :doc:`Replenishment report <../product_replenishment/strategies>`
+     - Yes
+     - :ref:`No* <inventory/product_management/replenish>`
+     - No
+   * - Can be manufactured or subcontracted
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - :ref:`Yes* <inventory/product_management/manufacture>`
+     - No
+   * - Can be in a kit
+     - Yes
+     - Yes
+     - No
+   * - Placed in package
+     - Yes
+     - :ref:`Yes* <inventory/product_management/package>`
+     - No
+   * - Appears on Inventory report
+     - Yes
+     - No
+     - No
+
+.. _inventory/product_management/on-hand-store:
+
+On-hand quantity
+----------------
+
+A storable product's on-hand and forecasted quantity based on incoming and outgoing orders are
```

Comment:
```suggestion
A storable product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
```
Since these are two separate quantities